recipes/default.rb

```diff
@@ -3,11 +3,11 @@
 #>
 
 package 'sensu-community-plugins' do
-  action :upgrade
+  action :remove
```

Comment:
We install gems instead of using the sensu-community-plugins package.

The gems are installed in sensu's embedded ruby, our package went into /opt/sensu-community-plugins